src/modules/vault/hooks/assets/useVaultAssets.ts

```diff
@@ -113,6 +113,7 @@ function useVaultAssets(predicate?: Vault) {
 
       return (
         bn(bn.parseUnits(balance.amount!))
+          //@ts-ignore - TODO: minGasPrice was removed in fuels@0.88.1
           .sub(gasConfig?.minGasPrice || bn.parseUnits('0.00001'))
```

Comment:
`minGasPrice` was removed in fuels@0.88.1

Reply:
use maxGasPerPredicate